Minor change to leaderboards.md: Clarification of game speed hint

akleemans opened the pull request:
**Edit**: Instead of removing the part, I suggest to improve it, see my latest comment below.

Original comment:

---

Remove confusing part of sentence.

I really struggled when reading the part I suggest to remove. What's the meaning of "see not below" - don't look below? Even if only the "not" was misplaced, there is nothing more below on the page about framerates or game speed.

I checked the [Initial commit](https://github.com/RetroAchievements/docs/blob/f7f48cd8661671079b9c6949a8070804d3f6a7bc/archive/Leaderboards.md) where this was added but was not able to find anything related to this.

Furthermore, a search across the RA docs with "framerate", "frame rate" or "game speed" does not return any page with additional information (except maybe for the [`ResetIf` docs](https://docs.retroachievements.org/developer-docs/real-examples/creating-a-timer-with-reset-if-hits-based-on-the-speed-of-the-game.html#creating-a-timer-with-resetif-hits-based-on-the-speed-of-the-game) with a short note on PAL vs. NTSC, but I'm not sure if it's worth linking it).

So my suggestion would be to remove this part of the sentence alltogether.

akleemans committed:
Update leaderboards.md

Remove confusing part of sentence

Jamiras commented:
The missing note below was probably documenting the formats, which now live [here](https://docs.retroachievements.org/developer-docs/rich-presence.html#format) (see the notes below the table).

> * `FRAMES` calculation takes the value, multiplies it by 100, and divides it by 60. If your framerate is something other than 60, you'll need to do the conversion yourself and use `MILLISECS`.

The issue the comment is trying to address is that using the FRAMES format always divides by 60 even if your game is running at 50fps (or even something else like 30fps).

akleemans commented:
After reading through the page again, I think I found what was meant: a not**e**, maybe the one at the subchapter [Value Format](https://docs.retroachievements.org/developer-docs/leaderboards.html#value-format):

> NOTE: If you're using a system that doesn't run at 60Hz, you have to use "Time (Centiseconds)" and multiply by some value to do the conversion yourself (50Hz = *2 [100/50], 60Hz = *1.666666 [100/60], 75Hz =*1.333333 [100/75]).

Not sure if a direct link to that paragraph can be added? There are several "note"s inbetween, so the reference to "not(e) below" is confusing.

akleemans commented:
@Jamiras Good point, so the comment should stay in some form. I think the note on Value Format is even more fitting? I'll update the PR later this evening to reference to it.

akleemans committed:
Update leaderboard documentation for frame rate details

Clarify the frame rate context for the leaderboard type.

akleemans committed:
Fix link in leaderboards documentation

Updated link in the Type section for clarity.

akleemans commented:
Updated PR:
* Added the missing "e" on "note"
* Added direct link to paragraph so it's clear which note is meant (instead of just "note below", there are several notes on this page)
* Moved "." to end (to close sentence after brackets instead of before)

akleemans commented:
@Jamiras What do you think of the edited PR?

Jamiras commented on docs/developer-docs/leaderboards.md:
```diff
@@ -55,7 +55,7 @@ The best place to start is to look at existing leaderboards and break it down to
 The **Title/Description** fields are quite obvious.
 
-The **Type** is "Time (Frames)". The value we're tracking updates once a frame, and the Genesis runs at 60 frames per second. (see not below for systems that run at other speeds)
+The **Type** is "Time (Frames)". The value we're tracking updates once a frame, and the Genesis runs at 60 frames per second (see note below on [Value Format](/developer-docs/leaderboards#value-format) for systems that run at other speeds).
```
The actual URL is https://docs.retroachievements.org/developer-docs/leaderboards.html#value-format. I don't know if the ".html" is implied and will be added by the conversion.

Jamiras replied:
I'm a bit leary about saying "see note below" when it's the note on the linked document. Maybe just "see note on...".

redwizard42 replied:
> The actual URL is https://docs.retroachievements.org/developer-docs/leaderboards.html#value-format. I don't know if the ".html" is implied and will be added by the conversion.

The style link he has will get built to the right url when it's built and deployed.

akleemans replied:
> I'm a bit leary about saying "see note below" when it's the note on the linked document. Maybe just "see note on...".

The link is to a paragraph far down below, but still on the same page (leaderboards.html), that's why I kept the `below` (which was there originally). But it can also be removed if it doesn't fit.

Jamiras reviewed (commented):
I think the changes address the previous problems. I've left a couple comments, but will leave final approval to someone more versed in managing the documentation repository.
